es_ES/LC_MESSAGES/archive.po

```diff
@@ -245,38 +245,38 @@ msgid "SPOTIFY"
 msgstr "Spotify"
 
 msgid "STARS"
-msgstr "Stars"
+msgstr "Estrellas"
 
 msgid "SUDOMEMO"
 msgstr "Sudomemo"
 
 msgid "THIS_FLIPNOTE_IS_SPINOFF_OF"
-msgstr "This Flipnote is a spin-off of"
+msgstr "Este Flipnote es un spin-off de"
 
 msgid "TOTAL_FLIPNOTES"
-msgstr "Total Flipnotes"
+msgstr "Flipnotes en total"
 
 msgid "USER"
-msgstr "User"
+msgstr "Usuario"
 
 msgid "USERS"
-msgstr "Users"
+msgstr "Usuarios"
 
 # Austin's Flipnote from Flipnote Hatena, May 17th, 2010
 msgid "USERS_ARCHIVED_FLIPNOTE_FORMAT"
-msgstr "%s's Flipnote from Flipnote Hatena, %s"
+msgstr "Flipnotes de %s de Flipnote Hatena, %s"
```

Comment:
Maybe "Flipnotes por %s" instead of de? Por implies it being the user's creation while de implies ownership to me.